app/_includes/plugins/hub-examples/consumer_groups.html

```diff
@@ -0,0 +1,74 @@
+{%- assign hub_example = include.hub_example -%}
+{%- assign kong_manager_fields = hub_example.kong_manager.fields -%}
+
+{% if include.render_intro_text %}
+The following examples provide some typical configurations for enabling
+the <code>{{ hub_example.plugin_name }}</code> plugin on a
+[consume group](gateway/latest/admin-api/consumer-groups/reference/).
+{% endif %}
+
+{% navtabs %}
+  {% if hub_example.render_curl? %}
+  {% navtab Admin API %}
+
+Make the following request:
+
+{% include_cached plugins/hub-examples/curl.html example=hub_example.curl %}
+
+Replace `CONSUMER_GROUP_NAME|CONSUMER_GROUP_ID` with the `id` or `name` of the consumer that this plugin configuration will target.
+
+{% if page.service_id and page.route_id %}
+You can combine `consumer.id`, `service.id`, or `route.id`
+{% elsif page.service_id %} and `service.id`
+{% elsif page.route_id %} and `route.id`
+{% endif %} in the same request, to further narrow the scope of the plugin.
```

Comment:
```suggestion
```
Let's remove this for now. It's broken in general, apparently:
![Screenshot 2023-08-03 at 11 03 28 AM](https://github.com/Kong/docs.konghq.com/assets/54370747/0cc62e49-8745-41d5-8c65-97b962cc806c)